test/utils/helpers.ts

```diff
@@ -201,4 +211,23 @@ export const bumpFeesForBtcTx = async (
     bitcoinCoreClient: BitcoinCoreClient,
     txId: string, 
     options: RbfOptions = {conf_target: 72, estimate_mode: RbfEstimateMode.economical}
-): Promise<RbfResponse> => bitcoinCoreClient.client.command("bumpfee", txId, options);
\ No newline at end of file
+): Promise<RbfResponse> => bitcoinCoreClient.client.command("bumpfee", txId, options);
+
+/**
+ * Wait for an event to show up as finalized
+ * @param interBtcApi the api to use for querying
+ * @param event the event to wait for, eg. `api.events.assetRegistry.RegisteredAsset`
+ */
+export const waitForFinalizedEvent = async <T extends AnyTuple>(
+    interBtcApi: InterBtcApi,
+    event: AugmentedEvent<ApiTypes, T>
+): Promise<void> => {
+    return new Promise<void>((resolve, _) => interBtcApi.system.subscribeToFinalizedBlockHeads(
+        async (header) => {
+            const events = await interBtcApi.api.query.system.events.at(header.parentHash);
```

Comment:
Why use the hash of the parent instead of the hash of the finalized block? If there is a reason for that then it'd be good to have a comment stating it